unfoldingWord/translationCore
# Added bible loading action to app.js

#### This pull request addresses:

Now copying all bibles from tC `static folder` to `user/USERNAME/translationCore/resources/bibles`


#### How to test this pull request:
- Make sure you don't have any Bible in `user/USERNAME/translationCore/resources/bibles` and open tC then verify all the bibles are in the aforementioned path.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1865)
